client/src/styles/Footer.css

```diff
@@ -1,23 +1,5 @@
 .footer {
-  display: flex;
-  position: fixed;
-  left: 0;
   bottom: 0;
   width: 100%;
-  background-color: black;
-  color: white;
-  text-align: center;
-  padding: 10px;
-}
-
-.footer-image {
-  max-width: 50px;
-  max-height: 50px;
-  flex: 1;
-}
-.team-container {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  flex: 1;
+  position: relative;
```

Comment:
Since these 3 properties don't do anything to the footer element, let's delete this file